crates/oxc_linter/src/snapshots/jsx_no_duplicate_props.snap

```diff
@@ -0,0 +1,73 @@
+---
+source: crates/oxc_linter/src/tester.rs
+expression: jsx_no_duplicate_props
+---
+  ⚠ eslint-plugin-react(jsx-no-duplicate-props): No duplicate props allowed. The prop "a" is duplicated.
+   ╭─[jsx_no_duplicate_props.tsx:1:1]
+ 1 │ <App a a />;
+   ·      ─ ─
+   ╰────
+  help: Remove one of the props, or rename them so each prop is distinct.
+
+  ⚠ eslint-plugin-react(jsx-no-duplicate-props): No duplicate props allowed. The prop "A" is duplicated.
+   ╭─[jsx_no_duplicate_props.tsx:1:1]
+ 1 │ <App A b c A />;
+   ·      ─     ─
+   ╰────
+  help: Remove one of the props, or rename them so each prop is distinct.
+
+  ⚠ eslint-plugin-react(jsx-no-duplicate-props): No duplicate props allowed. The prop "a" is duplicated.
+   ╭─[jsx_no_duplicate_props.tsx:1:1]
+ 1 │ <App a="a" b="b" a="a" />;
+   ·      ─────       ─────
```

Comment:
Shall we use the span `a`? The focus is on the key, not the value.

Reply:
yeah seems reasonable - eslint reports the whole prop.

but eslint also doesn't reference the first one